dawn/renderer/components/Editor.js

```diff
@@ -240,21 +320,21 @@ class Editor extends React.Component {
           <ButtonGroup id="code-execution-buttons">
             <EditorButton
               text="Run"
-              onClick={this.startRobot}
+              onClick={() => { this.startRobot.call(this); }}
```

Comment:
Better to be consistent and bind `this.startRobot` in the constructor (same for all the other cases below)